src/unk_0202631C.c

```diff
@@ -1252,29 +1260,29 @@ u32 sub_0202756C (const UnkStruct_02026324 * param0, int param1, int param2)
     return 0;
 }
 
-u32 sub_020276C8 (const UnkStruct_02026324 * param0, int param1)
+u32 sub_020276C8 (const UnkStruct_02026324 * pokedex, int param1)
```

Comment:
This looks like forms to me.

- 201 is Unown
- 422 is Shellos
- 423 is Gastrodon
- 412 is Burmy
- 413 is Wormadam
- 386 is Deoxys
- 492 is Shaymin
- 487 is Giratina
- 479 is Rotom

Reply:
Done, I added the relevant constants